monitoring/dashboards/viya/sas-launched-jobs-users.json

```diff
@@ -1419,7 +1419,9 @@
   "refresh": "10s",
   "schemaVersion": 31,
   "style": "dark",
-  "tags": [],
+  "tags": [
+    "sas-viya"
+  ],
   "templating": {
```

Comment:
The fact that there is only one change to this file makes me think we should have been able to achieve the same results on the other file with a single change.